modules/bankfeed/api/bankfeed.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+import BankData from '../data/bandata';
+
+module.exports = function (app: core.Express) {
+  // Get a list of bank statement
+  app.get('/', (req: Request, res: Response) => {
+    const response = { message: '', data: {} };
+    try {
+      if (req.method === 'get') {
```

Comment:
For success we like to just use res.json directly and res.status(400) for errors. 

See names-routes.ts for examples.